langlib/jballerina.java/src/main/java/org/ballerinalang/langlib/java/Cast.java

```diff
@@ -68,11 +68,10 @@ public static Object cast(BObject value, BTypedesc castType) {
             }
             Type describingBType = castType.getDescribingType();
             BString castObjClass;
-            ObjectType castObjType;
+            BObjectType castObjType;
```

Comment:
I think we should use `io.ballerina.runtime.api.types.ObjectType` (the runtime public API) instead of `BObjectType` here.

Reply:
Done